test/unit-tests/function/matrix/pinv.test.js

```diff
@@ -32,7 +32,7 @@ function assertValidPinv (A, A_) {
 }
 
 describe('pinv', function () {
-  function test (A, A_, strict = false) {
+  function check (A, A_, strict = false) {
     const pinvA = pinv(A)
```

Comment:
Because `test('description', cb)` is an alias for `it('description', cb)`, this was trigging [`no-nested-tests`](https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-nested-tests.md`.

Of course, it wasn't really a nested test since `test(...)` had been redefined. But re-defining `test` inside of mocha doesn't seem great anyway.

Reply:
👍